lib/web_ui/lib/src/engine/skwasm/skwasm_impl/raw/raw_memory.dart

```diff
@@ -140,6 +164,22 @@ class StackScope {
     return pointer;
   }
 
+  Pointer<Uint16> convertIntsToUint16Native(List<int> values) {
+    final Pointer<Uint16> pointer = allocUint16Array(values.length);
+    for (int i = 0; i < values.length; i++) {
```

Comment:
We should request bulk copy operations in FFI API.

Reply:
I think https://github.com/dart-lang/sdk/issues/43968 is an existing issue for this. That being said, for that to actually make any difference for the wasm backend, we would need https://github.com/WebAssembly/gc/issues/367 because otherwise it would just lower to the same thing I've written here.